README.adoc

```diff
@@ -1,102 +1,405 @@
 = Asciidoctor Browser Extension
 Guillaume Grossetie
-
+// Settings
+:table-frame: none
+:table-grid: none
+:table-stripes: none
+:experimental:
+// GitHub specific settings
 ifndef::env-github[:icons: font]
 ifdef::env-github[]
 :outfilesuffix: .adoc
 :caution-caption: :fire:
 :important-caption: :exclamation:
-:note-caption: :paperclip:
+:note-caption: :information_source:
 :tip-caption: :bulb:
 :warning-caption: :warning:
 endif::[]
-
+// AsciiDoc attributes
+:chromium-browser-list: Chrome, Edge, Chromium, Brave or Opera
+:project-name: Asciidoctor Browser Extension
+:uri-project-repo: https://github.com/asciidoctor/asciidoctor-browser-extension
 :uri-license: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/LICENSE
-:uri-hacking: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/hacking.adoc
+:uri-contributing: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/CONTRIBUTING.adoc
 :uri-changelog: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/changelog.adoc
+// Asciidoctor Projects and Docs
+:uri-asciidoc: https://asciidoc.org
+:uri-asciidoctor-docs: https://docs.asciidoctor.org
+:uri-asciidoc-docs: https://docs.asciidoctor.org/asciidoc/latest
+:uri-asciidoctor-js-repo: https://github.com/asciidoctor/asciidoctor.js
+// Webstores
 :uri-chrome-webstore: https://chrome.google.com/webstore/detail/asciidoctorjs-live-previe/iaalpfgpbocpdfblpnhhgllgbdbchmia
 :uri-mozilla-addon: https://addons.mozilla.org/firefox/addon/asciidoctorjs-live-preview
 :uri-opera-addon: https://addons.opera.com/extensions/details/asciidoctorjs-live-preview
 :uri-edge-webstore: https://microsoftedge.microsoft.com/addons/detail/asciidoctorjs-live-previ/pefkelkanablhjdekgdahplkccnbdggd
+// Graphics
+:uri-chrome-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png
+:uri-mozilla-addon-image: https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg
+:uri-opera-addon-image: https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png
+:uri-edge-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png
+:uri-asciidoctor-image: https://avatars.githubusercontent.com/u/3137042?s=200&v=4
+// IMPORTANT - Change the branch in the URIs below - This is for testing only
+:uri-ext-active-image: https://raw.githubusercontent.com/sturtison/asciidoctor-browser-extension/docs/issue-648-refresh-docs-draft/promotional/ext-active.png
+:uri-ext-inactive-image: https://raw.githubusercontent.com/sturtison/asciidoctor-browser-extension/docs/issue-648-refresh-docs-draft/promotional/ext-inactive.png
+
+
+image::{uri-asciidoctor-image}[Asciidoctor icon,role="related thumb left",height="50px",width="50px"]
```

Comment:
I don't think the Asciidoctor logo is needed here. This repository is already in the Asciidoctor organization, and this just takes up unnecessary space.

Reply:
I agree with the space. The image however loads from the Asciidoctor project avatar image on GitHub - https://avatars.githubusercontent.com/u/3137042?s=200&v=4

If you still think it's not needed because of the branding or overuse of the Asciidoctor logo, I'll remove it.

Reply:
I'll remove it - If you think it adds value to the user we can add the reference to Asciidoctor avatar back in.